test/mip_01.jl

```diff
@@ -1,6 +1,6 @@
 # an example on mixed integer programming
 #
-#   maximize x + 2 y + 5 z
+#   maximize x + 3 y + 5 z
```

Comment:
It didn't have a unique optimal solution, which is why the test started failing on a new version